reference/3.0/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -179,10 +179,47 @@ their type and value:
 |    482gb | Int64   | 517543559168 |
 | 0x1e2lgb | Int64   | 517543559168 |
 
-> [!NOTE]
-> Any commands that look like a numeric literal must be executed using the the
-> call operator, otherwise they are interpreted as a number of the associated
-> type.
+### Commands that look like numeric literals
+
+Any command that looks like a numeric literal must be executed using the the
+call operator (`&`), otherwise it is interpreted as a number of the associated
+type.
+
+### Access properties and methods of numeric objects
+
+To access a member of a numeric literals, there are cases when you need to
+enclose the literal in parentheses.
+
+- The literal does not have a decimal point
+- The literal does not have any digits following the decimal point
+- The literal does not have a suffix
+
+For example, the following example fails:
+
+```
+PS> 2.GetType().Name
+At line:1 char:11
++ 2.GetType().Name
++           ~
+An expression was expected after '('.
++ CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
++ FullyQualifiedErrorId : ExpectedExpression
+```
+
+The following examples work:
+
+```
+PS> 2l.GetType().Name
```

Comment:
This example doesn't work. And if it did, it would return Int32 and not Int64.

Reply:
This is an l, not 21 like I first read it.  Sean is going to upper case the L so others don't make the mistake I did while reviewing it.  Good after that update.